populateCouch.js

```diff
@@ -0,0 +1,54 @@
+require('dotenv').config();
+const nano = require('nano')(`http://admin:${process.env.NANOPZ}@localhost:5984`);
+
+let NANO_DB = 'test'
+let ID_START = 0
+let BATCHSIZE = 1000
+let BATCHLOOPS = 10000
+
+let makeDataArray = () => {
+  let arr = []
+  for (let i = 0; i < BATCHSIZE; i++) {
+    arr.push({'test': ID_START}) // INSERT DATA TEST HERE
+    ID_START++
+  }
+  return arr
+}
+
+let insertArray = async (dbArray) => {
+  try {
+    const nanodb = nano.use(NANO_DB)
```

Comment:
It might be worth defining nanodb as a global variable, as using it here would generate an extra pointer to it for every insertArray instance running